catalog_tools/download/download_catalogs.py

```diff
@@ -20,14 +21,14 @@ def apply_edwards(mag_type: str, mag: float) -> pd.Series:
         return pd.Series([mag_type, mag])
 
 
-def download_catalog_sed(
-        start_time: dt.datetime = dt.datetime(1970, 1, 1),
```

Comment:
Below you use pd.to_datetime - not sure if this results in the same datatype (dt.datetime). I think we should stay consistent and only use one time type